src/repl/helper.rs

```diff
@@ -0,0 +1,71 @@
+use rustyline::{
+    completion::{Completer, Pair},
+    error::ReadlineError,
+    highlight::{Highlighter, MatchingBracketHighlighter},
+    hint::{Hinter, HistoryHinter},
+    validate::{self, MatchingBracketValidator, Validator},
+    Context,
+};
+use rustyline_derive::Helper;
+use std::borrow::Cow;
+
+#[derive(Helper)]
+pub(super) struct ReplHelper {
+    pub(super) highlighter: MatchingBracketHighlighter,
+    pub(super) validator: MatchingBracketValidator,
```

Comment:
Hmm, this seems ok for the very first draft, but eventually I'd like to implement proper syntax highlighting.

Reply:
Since there's also #389, and probably other use cases for syntax highlighting, a generic module for syntax highlighting would be a good idea.